salt/modules/aptkernelpkg.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+'''
+Manage Linux kernel packages on APT-based systems
+'''
+from __future__ import absolute_import
+import logging
+import re
+
+# Import 3rd-party libs
+try:
+    from salt.utils.versions import LooseVersion as _LooseVersion
+    from salt.ext.six.moves import filter  # pylint: disable=import-error,redefined-builtin
+    HAS_REQUIRED_LIBS = True
+except ImportError:
+    HAS_REQUIRED_LIBS = False
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'kernelpkg'
+
+
+def __virtual__():
+    '''
+    Load this module on Debian-based systems only
+    '''
+
+    if not HAS_REQUIRED_LIBS:
+        return (False, "Required library could not be imported")
+
+    if __grains__.get('os_family') in ('Kali', 'Debian'):
```

Comment:
Any reason why you do not specify a default on this call like you did on the one on line 33?